aip/0123.md

```diff
@@ -69,7 +69,12 @@ message Topic {
 }
 ```
 
+- Patterns **must** correspond to the [resource name][aip-122].
+- Pattern variables (the stuff within braces) **must** use `snake_case`, and
```

Comment:
"segments" instead of "stuff"? "identifiers"? Or at least something else?

Reply:
```suggestion
- Pattern variables (the segments within braces) **must** use `snake_case`, and
```